rpc/backend/blocks.go

```diff
@@ -283,6 +284,51 @@ func (b *Backend) EthMsgsFromTendermintBlock(
 	return result
 }
 
+func (b *Backend) EthMsgsFromTendermintEndBlock(
```

Comment:
can method reflect that this is pulling messages from events instead?
is msgs correct here, i think msgs are used in cosmos for triggering transactions in modules, is what we pull from events here msg in that sense or we just read some struct from event?

Reply:
Yes we read the same struct as the txs EthMsg